ddtrace/pin.py

```diff
@@ -35,9 +35,10 @@ class Pin(object):
         >>> conn = sqlite.connect('/tmp/image.db')
     """
 
-    __slots__ = ["app", "tags", "tracer", "_target", "_config", "_initialized"]
```

Comment:
do we need a removed property for `Pin.app`?

Reply:
+1 to this. Looks like `Pin.app` will now be an `AttributeError`, good catch @brettlangdon!